### DISCUSSION TITLE
Using the FixedSizeList preventing the resize of columns on the table

### DISCUSSION THREAD

**Discussion by yossiHav** [TanStack/table#2376] [General]
First great library,

I've tried using the FixedSizeList , after reviewing your example for this issue, it worked great but the resize is working only on the header but not on the cells. 

Thanks

**Comment by yossiHav** [comment]
I found the problem. The callback hook is not rendering when resizing , the resizing change is not caught when listing to the prepareRow, any idea to solve this issue?

**Comment by sumitkr003** [comment]
Hi guys, i am also having this issue.
any luck ?

**Comment by yossiHav** [comment]
you need to the cell props on the useCallback, so each time the size of a cell is changing it will render row/column
